linux-4.4.defconfig

```diff
@@ -78,7 +78,6 @@ CONFIG_SYN_COOKIES=y
 # CONFIG_IPV6 is not set
 CONFIG_CFG80211=y
 CONFIG_CFG80211_INTERNAL_REGDB=y
-CONFIG_CFG80211_WEXT=y
```

Comment:
I'm not sure whether removing this option will inhibit other wifi device drivers from working.